src/core_codemods/sql_parameterization.py

```diff
@@ -34,7 +34,7 @@
 
 
 class SQLQueryParameterization(BaseCodemod, UtilsMixin, Codemod):
-    SUMMARY = "Parameterize SQL queries."
+    SUMMARY = "Parameterize SQL Queries"
```

Comment:
I see no reason for the capitalization here.

Reply:
This becomes the title of the PR and the docs, which are capitalized for consistency.